setup.py

```diff
@@ -95,25 +96,38 @@ def get_version(version_file):
         "boto3 == 1.21.10",
         "botocore == 1.24.10",
         "chevron == 0.14.0",
-        "docopt == 0.6.2",
+        "celery",
+        "click==8.0.3",
+        "docopt",
         "glob2 == 0.7",
-        "importlib-resources == 5.4.0",
+        "flask",
+        "flask_wtf",
+        "flask_login",
+        "flask_migrate",
+        "Flask-SQLAlchemy",
```

Comment:
Maintain alphabetical order:
```suggestion
        "flask",
        "flask_login",
        "flask_migrate",
        "flask_wtf",
        "Flask-SQLAlchemy",
```